config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Exhibits::Application.routes.draw do
   # API compatible with is_it_working checks
   match "/is_it_working" => "ok_computer/ok_computer#index", via: [:get, :options]
-  mount OkComputer::Engine, at: "/is_it_working"
+  match "/status" => "ok_computer/ok_computer#index", via: [:get, :options]
```

Comment:
Do we need to explicitly do this here, or are we back on the happy path:

https://github.com/sportngin/okcomputer#changing-the-okcomputer-route